<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxPort"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFlowControl">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware (RTS/CTS)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Software (XON/XOFF)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutChart"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditRead"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditFilePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSendFile">
        <property name="text">
         <string>Send File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRecieveFile">
        <property name="text">
         <string>Recieve File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="lineEditSend"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
